### PR TITLE
Add user-specific and developer-specific documentation (in GitHub README files, website)

### LINKED ISSUE #704: Documentation, mixs directory, files
At the top level of mixs GitHub:
     to develop a .md documentation file to explain the upper level structure of the directory.

To explain the - end user directories and the developer directories. 

To document here the plan for devising this documentation.

### LINKED ISSUE #807: Create list of documents supporting or clarifying MIxS
Eventually to be added as `see_also` annotations on the MIxS Schema Definition

### PR DESCRIPTION
**Pull request by sujaypatil96**
- [x] Update [Repository Structure](https://github.com/GenomicsStandardsConsortium/mixs/blob/main/README.md#repository-structure) section in [README.md](https://github.com/GenomicsStandardsConsortium/mixs/blob/main/README.md)
- [ ] 

Fixes #704

### CODE REVIEW

**Comment by sujaypatil96** [comment]
@turbomam we can collaborate on documentation updates on this branch. Feel free to suggest/make modifications as you see fit and we can work on it together.

File: README.md [hunk 1]
```diff
@@ -1,39 +1,33 @@
 # MIxS: Minimum Information about any (X) Sequence
 
-⚠️ ⚠️ ⚠️ 
+MIxS is a specification for the Minimum Information about any (X) Sequence standard.
```
Comment (turbomam) [inline README.md:3]: This sounds more like X and Y are synonyms. So what is MIxS really a specification for? That is an extremely tricky question.

Comment (turbomam) [reply README.md:3]: I would suggest something more like

MIxS, or the Minimum Information about any (X) Sequence is a specification for describing the provenance of a nucleotide sequence:
- what do we know about the sample it came from?
- in what environment was the sample found?
- in what ways were the nucleic acids extracted, amplified, etc from the sample?

File: README.md [hunk 2]
```diff
@@ -1,39 +1,33 @@
...
 
-The new release primarily consists of converting a spreadsheet expressing MIxS 6.1 into a [LinkML artifact, which now serves as the source of truth](https://github.com/GenomicsStandardsConsortium/mixs/blob/main/src/mixs/schema/mixs.yaml). 
-We have also started to follow the [LinkML cookiecutter layout](https://github.com/linkml/linkml-project-cookiecutter).
-MIxS users should see very few changes in the content, although legacy term attributes like "Definition", "Expected value", "Value syntax" and "Section" have been broken down into their LinkML equivalents, 
-like `description`, `range`, `pattern` and `in_subset`.
+The public-facing web documentation site can be browsed at: https://genomicsstandardsconsortium.github.io/mixs/
 
-We have [some known issues](https://github.com/GenomicsStandardsConsortium/mixs/issues?q=is%3Aissue+is%3Aopen+label%3A6.1.1-%3E6.2.0), and you may find others. [Please report them](https://github.com/GenomicsStandardsConsortium/mixs/issues) including text snippets and or screen shots!
+MIxS is divided into Checklists and Extensions (previously called Packages).
```
Comment (turbomam) [inline README.md:9]: what does "divided into mean"

I would suggest: Most MIxS users will primary relate to MIxS term, which describe the samples and their sequences. But terms aren't free-standing. They belong to Checklists, Extensions (...) and combinations of those two. That means we can say "for this kind of sample, you mus tell us bout..."

File: README.md [hunk 3]
```diff
@@ -1,39 +1,33 @@
...
 
-If you have questions or concerns in the interim, please contact gensc-twg@groups.google.com
+[Extensions](https://genomicsstandardsconsortium.github.io/mixs/#extensions) supplement Checklists by providing additional terms to describe specific environments from which a sample was collected. For example, the Agriculture Extension has a number of terms to describe agricultural environments.
 
-⚠️ ⚠️ ⚠️ 
-
-## Documentation Website
-
-[https://w3id.org/mixs](https://w3id.org/mixs)
+Extensions can be used in conjunction with any Checklist. We call them [Combinations](https://genomicsstandardsconsortium.github.io/mixs/combinations/). For example, if you have sequenced a metagenome sample retrieved from an agricultural environment, then you would be able to use a Combination of the [Mims](https://genomicsstandardsconsortium.github.io/mixs/0010007/) Checklist and [Agriculture](https://genomicsstandardsconsortium.github.io/mixs/0016018/) Extension, called [MimsAgriculture](https://genomicsstandardsconsortium.github.io/mixs/0010003_0016018/).
```
Comment (turbomam) [inline README.md:15]: who's we and what's them?

Comment (turbomam) [reply README.md:15]: MIxS also contains Combinations(link) of Extensions and Checklists. ?

File: README.md [hunk 4]
```diff
@@ -1,39 +1,33 @@
...
-* [examples/](examples/) - example data
-* [project/](project/) - project files (do not edit these)
-* [src/](src/) - source files (edit these)
-  * [mixs](src/mixs)
-    * [schema](src/mixs/schema) -- LinkML schema
-      (edit this)
-    * [datamodel](src/mixs/datamodel) -- generated
-      Python datamodel
-* [tests/](tests/) - Python tests
+Below is documentation about the contents of the various user-facing folders/directories in this repository.
```
Comment (turbomam) [inline README.md:19]: what do you mean documentation about? I think we could be much terser?

**Review by lschriml — COMMENTED** [review]
Let's go over this file in the next CIG call.

**Review by turbomam — APPROVED** [review]
I approve this but don't object to more discussion at a Tuesday morning meeting.

**Review by lschriml — APPROVED** [review]
minor edits

**Pull request merged**